shared/src/model/exercise.js

```diff
@@ -95,4 +95,15 @@ class Exercise extends BaseModel {
   @computed get isPublishable() {
     return Boolean(!this.isNew && this.validity.valid && !this.published_at);
   }
+
+  @action onQuestionFreeResponseSelected() {
+    if (this.isMultiPart) { return; }
```

Comment:
This return seems odd to me. Are multipart WRQs allowed elsewhere?

Reply:
yeah it also seemed odd to me but that's exactly what the card read and that's why I asked about practice 😁